lib/pages/chat_list/receive_sharing_intent_mixin.dart

```diff
@@ -47,10 +47,14 @@ mixin ReceiveSharingIntentMixin<T extends StatefulWidget> on State<T> {
 
   void _processIncomingUris(String? text) async {
     if (text == null) return;
-    TwakeApp.router.go('/share');
-    WidgetsBinding.instance.addPostFrameCallback((_) {
-      UrlLauncher(context, text).openMatrixToUrl();
-    });
+    if (text.startsWith(AppConfig.registrationSchemePrefix)) {
+      matrixState.handleRegistrationDone(text);
```

Comment:
why need `matrixState`?

Reply:
i put the `handleRegistrationDone` in a mixin and the MatrixState use it